test/integration/targets/collections_internal_redirect/collections/ansible_collections/testns/testcoll/plugins/modules/ping.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python
```

Comment:
any reason this needs to be a "real" copy of the ping module vs just a dummy module? (see the collections integration tests, there's a ton of dummy modules in there)